src/libraries/System.Runtime.Numerics/src/System/Number.BigInteger.cs

```diff
@@ -741,6 +741,7 @@ internal static bool TryFormatBigInteger(BigInteger value, ReadOnlySpan<char> fo
             return spanSuccess;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
Isn't this an enormous function to inline, and given the call chains, isn't it likely that all of this code will end up being inlined into the user's call sites to methods like BigInteger.ToString? This seems very undesirable.

Reply:
Oh, wait, nevermind, I read it as AggressiveInlining, not AggressiveOptimization.